### DISCUSSION TITLE
How to turn off 'Anti-aliasing filter'

### DISCUSSION THREAD

**Discussion by sojuo** [sylikc/jpegview#349] [Q&A] [answered]
How can I display images without anti-aliasing like [ImageGlass](https://github.com/d2phap/ImageGlass), since anti-aliasing is automatically enabled when displaying images?

https://github.com/user-attachments/assets/0e68f2ba-c7d1-45bd-8158-2d147d746a08

https://github.com/user-attachments/assets/8f25d76f-33a5-48d7-8e17-25816bf25741

**Comment by sylikc** [comment]
> How can I display images without anti-aliasing like [ImageGlass](https://github.com/d2phap/ImageGlass), since anti-aliasing is automatically enabled when displaying images?

I recall adding an option to the INI awhile back to disable either when scaling up or down for specific use cases

**Reply by sojuo** [reply]
> > How can I display images without anti-aliasing like [ImageGlass](https://github.com/d2phap/ImageGlass), since anti-aliasing is automatically enabled when displaying images?
> 
> I recall adding an option to the INI awhile back to disable either when scaling up or down for specific use cases

I tried to change the parameters of `DownSamplingFilter` and `Sharpen`, but it doesn't seem to work. Is there any way to make JPEGView display the image as it is without any modification?

**Reply by sojuo** [reply, marked as answer]
> > > How can I display images without anti-aliasing like [ImageGlass](https://github.com/d2phap/ImageGlass), since anti-aliasing is automatically enabled when displaying images?
> > 
> > 
> > I recall adding an option to the INI awhile back to disable either when scaling up or down for specific use cases
> 
> I tried to change the parameters of `DownSamplingFilter` and `Sharpen`, but it doesn't seem to work. Is there any way to make JPEGView display the image as it is without any modification?

Finally, I found the solution at https://github.com/sylikc/jpegview/issues/19#issuecomment-1455124096
Set HighQualityResampling to false to see it properly. But I don't know if there are any other modifications to the JPEGView. If so, please let me know.
